ui/models/job.js

```diff
@@ -116,36 +116,79 @@ export default class JobModel {
   }
 
   static async retrigger(jobIds, repoName, getGeckoDecisionTaskId, notify) {
-    const jobTerm = jobIds.length > 1 ? 'jobs' : 'job';
-
+    const jobTerm = Object.keys(jobIds).length > 1 ? 'jobs' : 'job';
     try {
       notify(`Attempting to retrigger/add ${jobTerm} via actions.json`, 'info');
+      const jobs = await JobModel.getList(repoName, { id__in: Object.keys(jobIds).join() });
+      // Add counts for each requested job
+      for (const job of jobs) {
+        job.count = jobIds[job.id];
+      }
+
+      // TODO: If two distinct jobs in a push having the same taskname are requested,
+      // taskcluster won't include it multiple times, so I need to do something else
+      // to multiply the request count for these jobs.
 
-      const jobs = await JobModel.getList(repoName, { id__in: jobIds.join() });
+      // TODO: If `count` is greater than the action-task-defined limit, I don't know what to do.
```

Comment:
I think I'd be tempted to do the max number but send a ``notify`` saying they exceeded the max by # amount.